idkit/src/services/walletconnect.ts

```diff
@@ -173,6 +176,7 @@ const buildVerificationRequest = (config: IDKitConfig) => ({
 			signal: generateSignal(config.signal).digest,
 			action_description: config.action_description,
 			external_nullifier: generateExternalNullifier(config.app_id, config.action).digest,
+			credential_types: config.credential_types?.length ? config.credential_types : [CredentialType.Orb],
```

Comment:
Should we be setting this default in the store instead? Feels v dirty to have here, data should have been valid wayyy before it gets here